css/design_system/components/nav.scss

```diff
@@ -1,55 +1,54 @@
-:root {
-  --nav-height: 4rem;
-}
-
 /**
  * @title Primary Nav
  * @category components
- * @status early draft
- * @description Navigation component w/ primary and secondary variants.
+ * @component_type nav
+ * @description This navigation component can be used with or without the menu.
  * @example
- * <nav class="nav nav-primary">
- *   <ul>
- *     <li>Nav element 1</li>
- *     <li>Nav element 2</li>
+ * <nav class="s-nav">
+ *   <a class="s-nav-icon" href="/" aria-label="Home">
+ *     <img class="s-nav-icon" src="https://assets.turing.edu/turing-school-mark-gray.png" alt="" />
+ *   </a>
+ *   <ul class="s-nav-menu">
+ *     <li>Link One</li>
+ *     <li>Link Two</li>
+ *     <li>Link Three</li>
  *   </ul>
  * </nav>
  */
-.nav {
-  font-weight: var(--s-font-weight-bold);
-  height: var(--nav-height);
+
+.s-nav {
+  @extend .s-shadow-sm;
+  @extend .s-py-2;
+  @extend .s-px-4;
+  @extend .s-font-bold;
+  letter-spacing: 0.75px;
   position: sticky;
-  width: 100%;
   z-index: 99;
-  margin-bottom: 0.5rem;
-}
-
-.nav ul {
-  list-style-type: none;
   display: flex;
-  flex-direction: row;
-  justify-content: space-between;
   align-items: flex-end;
-}
+  justify-content: space-between;
+  align-items: center;
 
-.nav ul li {
-  text-align: center;
-  padding: 1rem;
-}
+  .s-nav-icon {
+    height: 50px;
+    width: 50px;
+  }
 
-.nav ul li a {
-  color: var(--color-primary);
-  text-transform: uppercase;
-}
+  .s-nav-menu {
+    list-style: none;
+    display: flex;
+    @extend .s-mb-1;
```

Comment:
For this class and its child elements I think it's actually ok / expected to target specific tags because we want to enforce some semantic HTML (and also because we implicitly target lists with the `list-style` property). So I think using the ul/li tags as targets works. Also I think we don't need flex because all the elements are aligned in inline order. Here's an alternative that I think works just the same:

```scss
  ul.s-nav-menu {
    list-style: none;
    @extend .s-m-0;

    li {
      display: inline-block;
      @extend .s-text-uppercase;
      @extend .s-ml-3;
      @extend .s-mt-2; // to center the text given the added height from the border
      @extend .s-border-bottom;
      @extend .s-border-width-2;
      border-color: transparent;

      &:hover, &:active {
        @extend .s-border-cyan-400;
        cursor: pointer;
      }
    }
  }
```